### DISCUSSION TITLE
Reolink RTSP

### DISCUSSION THREAD

**Discussion by nthulke** [blakeblackshear/frigate#7317] [Ask A Question] [answered]
Hello, I have just started using Frigate and love it (moved over from ZM once I got my coral m.2 working) and I have 4 wyze cam 2 generic IP cam and and old tplink nc200. All works well but the tplink not having rtsp really chews up CPU for the HTTP stream. So I have been reading up and for what I need the Reolink looks like a cheap easy fit, but all the doco states to use reolink in HTTP mode. 

Why would I do that when the Reolink has the option for RTSP? Is there an issue with their RTSP feed and Frigate? 

Just want to know was I only want to use RTSP moving forward for any cameras as sll the other cams use 3-5% cpu where the HTTP camera sits at 50%-70%.

Cheers!

**Comment by NickM-27** [comment, marked as answer]
There's a lot to unpack here. First off you are probably looking at http mjpeg feed for the current camera which is higher cpu usage.

The reolinks have http-flv which is different and does not have higher CPU usage.

> Why would I do that when the Reolink has the option for RTSP? Is there an issue with their RTSP

Yes, reolinks RTSP is awful and contains visual as well as integrity issues. This is true across the board regardless of if you are using frigate or not

**Reply by nthulke** [reply]
Ahh ok thanks I'll do some more reading into alternatives then.
